src/trading/ordersFetcher.se

```diff
@@ -43,16 +43,16 @@ def getOrderIDs(type: int256, market: address, outcome: int256, startingOrderID:
     return(slice(orderIDs, items=0, items=i): arr)
 
 # @public
-def getOrder(orderID: address, type: int256, market: address, outcome: int256):
+def getOrder(_orderId: uint256, _type: uint256, _market: address, _outcome: uint256):
```

Comment:
Note to whoever migrates this to Solidity (@adrake33?).  Type should be an enum with two options and `_outcome` should be a `uint8`.  Serpent doesn't support either of these, so that change will have to wait.